pkgs/development/python-modules/arelle/test-cli.nix

```diff
@@ -0,0 +1,117 @@
+# Run with:
+#   cd nixpkgs
+#   nix-build -A python3Packages.arelle.passthru.tests.cli
+#
+# Note: These are uninteresting generated smoke tests to verify basic functionality
+{
+  lib,
+  runCommand,
+  arelle,
+}:
+
+runCommand "arelle-test-cli${lib.optionalString (!arelle.hasGUI) "-headless"}"
+  {
+    nativeBuildInputs = [ arelle ];
+  }
+  ''
+    # Set up temporary home directory
+    export HOME=$(mktemp -d)
+
+    # Test basic CLI commands work with proper assertions
+    arelleCmdLine --version --disablePersistentConfig 2>&1 | grep "Arelle(r) 2.37.59" > /dev/null
```

Comment:
Just realized: the version number is hardcoded here.
Shouldn't it have been substituted with the actual version?